client/src/routes/+page.svelte

```diff
@@ -1,13 +1,145 @@
 <script>
+    import { onMount } from 'svelte';
+
     const locationImg = new URL('../../static/location.svg', import.meta.url).href;
-    const categoryImg = new URL('../../static/burger.svg', import.meta.url).href;
+    const lockImg = new URL('../../static/lock.svg', import.meta.url).href;
     const priceImg = new URL('../../static/dollar.svg', import.meta.url).href;
 
-    function createSession() {
-        alert("this works");
+    // Food categories to pick from
+    let foodCategories = [ "New American",
+"Italian",
+"French",
+"Restaurants",
+"American",
+"Steakhouses",
+"Seafood",
+"Gastropubs",
+"Diners",
+"Bars",
+"Southern",
+"Wine Bars",
+"Mediterranean",
+"Cocktail Bars",
+"Spanish",
+"Nightlife",
+"Comfort Food",
+"Tapas/Small Plates",
+"Persian/Iranian",
+"Caribbean",
+"Mexican",
+"Portuguese",
+"Burgers",
+"Breakfast & Brunch",
+"Soul Food",
+"Lounges",
+"Pizza",
+"Peruvian",
+"Beer Bar",
+"Greek",
+"Food Court",
+"Cuban",
+"Pubs",
+"Brewpubs",
+"Latin American",
+"New Mexican Cuisine",
+"Thai",
+"Sports Bars",
+"Hawaiian",
+"Irish Pub",
+"Asian Fusion",
+"Pasta Shops",
+"Pop-Up Restaurants",
+"Cajun/Creole",
+"Salad",
+"Barbeque",
+"Tacos",
+"Indian",
+"Chinese",
+"Filipino",
+"Brazilian",
+"Fish & Chips",
+"Breweries",
+"Korean",
+"Vegetarian",
+"Middle Eastern",
+"Tex-Mex",
+"Buffets",
+"Vegan",
+"Gluten-Free",
+"Cafes",
+"Cafeteria",
+"Seafood Markets",
+"Sandwiches",
+"Teppanyaki",
+"Halal",
+"Vietnamese",
+"Wraps",
+"Himalayan/Nepalese",
+"Ramen",
+"Soup",
+"Specialty Food",
+"Desserts",
+"Sushi Bars",
+"Chicken Wings",
+"Japanese",
+"Hot Dogs",
+"Food Stands",
+"Food Delivery Services",
+"Creperies",
+"Delis",
+"Laotian",
+"Pancakes",
+"Waffles",
+"Food",
+"Chicken Shop",
+"Poke",
+"Street Vendors",
+"Hot Pot",
+"Noodles",
+"Fast Food",
+"Food Trucks",
+];
```

Comment:
Although we no longer need the food categories, please add the categories to a separate file so that we may import it (if needed) to declutter +page.svelte.

Reply:
Moved to components